frontend/www/problems/random.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace OmegaUp;
+require_once(dirname(__DIR__, 2) . '/server/bootstrap.php');
+if (OMEGAUP_LOCKDOWN) {
+    header('Location: /arena/');
+    die();
+}
+
+\OmegaUp\UITools::render(
+    fn (\OmegaUp\Request $r) => \OmegaUp\Controllers\Problem::getRandomProblemForSmarty(
+        $r
+    )
+);
```

Comment:
esto se puede reemplazar por un [HTTP 303](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/303):

```suggestion
$alias = \OmegaUp\DAO\Problems::getRandomProblemAlias();

header('HTTP/1.1 303 See Other');
header("Location: /arena/problem/$alias/");
```

no hay necesidad de desplegar una pantalla completa para sólo mostrar un enlace.